packages/legend-query-builder/src/stores/milestoning/QueryBuilderMilestoningState.ts

```diff
@@ -63,23 +63,35 @@ export class QueryBuilderMilestoningState implements Hashable {
   // TODO: Change this when we modify how we deal with milestoning.
   // See https://github.com/finos/legend-studio/issues/1149
   businessDate?: ValueSpecification | undefined;
+  businessDateValue?: ValueSpecification | undefined;
   processingDate?: ValueSpecification | undefined;
+  processingDateValue?: ValueSpecification | undefined;
 
   startDate?: ValueSpecification | undefined;
+  startDateValue?: ValueSpecification | undefined;
   endDate?: ValueSpecification | undefined;
+  endDateValue?: ValueSpecification | undefined;
 
   constructor(queryBuilderState: QueryBuilderState) {
     makeObservable(this, {
       processingDate: observable,
+      processingDateValue: observable,
```

Comment:
any reason we need these additional properties on this state. seems like we can jut calculate them in the component itself. 
Might become an issue to keep these in sync with the actual value of the milestoning date ?

Reply:
This is not necessary. I deleted these code.